Neos.Flow/Configuration/Objects.yaml

```diff
@@ -305,6 +305,15 @@ Neos\Flow\Property\PropertyMapper:
           1:
             value: Flow_PropertyMapper
 
+#
+# Reflection
+#
+Neos\Flow\Reflection\ReflectionService:
+  factoryObjectName: Neos\Flow\Reflection\ReflectionServiceFactory
+
+Neos\Flow\Reflection\ReflectionServiceFactory:
```

Comment:
The ReflectionServiceFactory already has a singleton annotation. What is that configuration for?